package.json

```diff
@@ -27,7 +27,7 @@
     "electron": "electron lib/backend/main.js",
     "electron:debug": "cross-env NODE_ENV=development electron lib/backend/main.js",
     "lint": "eslint --config package.json --no-eslintrc \"./src/**/*.{ts,tsx}\" 1>&2",
-    "start": "npm run build:backend && run-p \"start:frontend\" \"electron:debug\"",

```

Comment:
@williamkbentley why was this change made? With removing the `debug` you now aren't running in dev mode which is what the `start:frontend` script sets up for.